fast64_internal/mk64/f3d/properties.py

```diff
@@ -0,0 +1,43 @@
+from bpy.props import StringProperty, EnumProperty, FloatProperty, BoolProperty
+from bpy.types import PropertyGroup, UILayout
+from bpy.utils import register_class, unregister_class
+from ...utility import prop_split
+from ...render_settings import (
+    on_update_render_settings,
+)
+from ...f3d.f3d_material import ootEnumDrawLayers
+
+
+class MK64CourseDLImportSettings(PropertyGroup):
+    name: StringProperty(name="Name")
+    path: StringProperty(name="Directory", subtype="FILE_PATH")
+    basePath: StringProperty(name="Directory", subtype="FILE_PATH")
+    scale: FloatProperty(name="F3D Blender Scale", default=100, update=on_update_render_settings)
+    removeDoubles: BoolProperty(name="Remove Doubles", default=True)
+    importNormals: BoolProperty(name="Import Normals", default=True)
+    enableRenderModeDefault: BoolProperty(name="Set Render Mode by Default", default=True)
+
+    def draw_props(self, layout: UILayout):
+        prop_split(layout, self, "name", "Name")
+        prop_split(layout, self, "path", "File")
+        prop_split(layout, self, "basePath", "Base Path")
+        prop_split(layout, self, "scale", "Scale")
+        layout.prop(self, "removeDoubles")
+        layout.prop(self, "importNormals")
+
+        prop_split(layout, self, "enableRenderModeDefault", "Enable Render Mode by Default")
```

Comment:
```suggestion
        layout.prop(self, "enableRenderModeDefault")
```

Reply:
What is the difference ?

Reply:
One draws a split with a label and a prop with not text, the other draws a prop, most booleans are normal props in fast64 (because it looks better)